### ISSUE TITLE
Show a tab key symbol near a suggestion

### ISSUE THREAD

**Issue by shockone** [shockone/black-screen#550]
In order to avoid confusion as of what key needs to be pressed.

**Comment by unluisco** [comment]
Why we can't press the enter key to use the suggestion?

**Comment by shockone** [comment]
I tried it, and it's even worse. What should happen when we press enter here? 
![](https://dl.dropboxusercontent.com/spa/dlqheu39w0arg9q/e8lrtw3r.png)

**Comment by unluisco** [comment]
with both (tab and enter) should use the suggested option

in that case: cd src/main/

**Comment by shockone** [comment]
So, how do I execute `cd src/`?

**Comment by unluisco** [comment]
press Esc, like in atom

**Comment by shockone** [comment]
Exactly. We'd need to press two buttons in order to execute a command. Furthermore: sometimes it'd be one button, and sometimes two. From my experience, it was very confusing, dangerous, and counterintuitive. I think it's much better to have a simple rule that tab always completes, and return always executes.

**Comment by ekaragodin** [comment]
I strongly agree with @shockone. For example autocomplete in iTerm2 version 3 is very annoying.
